torch/nn/modules/module.py

```diff
@@ -1038,12 +1075,16 @@ def load_state_dict(self, state_dict: Union[Dict[str, Tensor], Dict[str, Tensor]
         missing_keys = []
         unexpected_keys = []
         error_msgs = []
+        state_dict = self._remove_prefix_from_state_dict_if_exists(state_dict, prefix='module.')
```

Comment:
Shouldn't this be `_strip_prefix_from_state_dict_if_exists` instead?

Reply:
Yes. I had changed the name initially when I started working it on this issue. When I rebased it changed stuff around. Sorry about that.

Reply:
Renamed everything to be consistent with `_strip_prefix_from_state_dict_if_exists`.